### DISCUSSION TITLE
Storybook v6.0.0 Release Date

### DISCUSSION THREAD

**Discussion by jetmir-mozenda** [storybookjs/storybook#11047] [General] [answered]
Are there any plans on when Storybook v6.0.0 is anticipated to be released?

**Comment by marjorg** [comment, marked as answer]
#9311 currently has it estimated to July 31st

**Reply by shilman** [reply]
FYI it was originally targeted for April, and has been slipping as the release grows in scope (and in the context of the pandemic). But the new estimate should give us time to wrap up everything that's currently on the plate--with any luck we'll be pushing those dates FORWARD if we can wrap things up sooner.

**Reply by marjorg** [reply]
@shilman I'm trying to find a migration guide to try out the beta, is that available yet?

**Reply by shilman** [reply]
Currently working on a "user-friendly" guide but in the meantime this is an exhaustive list of breaking changes: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-53x-to-60x

**Reply by jetmir-mozenda** [reply]
> FYI it was originally targeted for April, and has been slipping as the release grows in scope (and in the context of the pandemic). But the new estimate should give us time to wrap up everything that's currently on the plate--with any luck we'll be pushing those dates FORWARD if we can wrap things up sooner.

Any updates on the release date? Is tomorrow still the release date for v6.0.0?

**Reply by shilman** [reply]
Updated to Aug 10! Feeling good about that target. You can see our full set remaining issues here, none of which are hard blockers IMHO: https://github.com/storybookjs/storybook/milestone/15
